pipelines.adoc

```diff
@@ -173,7 +189,7 @@ The following steps explain how to run pipelines against custom built applicatio
 
 === Set up a container registry URL for the custom application stack image
```

Comment:
Change to
=== Setting up a container registry URL for the custom application stack image